setup.py

```diff
@@ -149,7 +149,12 @@ def run(self):
         'ipywidgets_bokeh',
         'ipyvolume',
         'ipyleaflet',
-        'xarray <0.17'
+        'xarray <0.17',
+        'pydeck',
+        'pyecharts',
+        'idom',
+        'xlsxwriter',
+        'pyarrow',
```

Comment:
Most of these cannot be added to setup.py since that is also the ground truth for conda installs. If I remove these here will this break the binder setup?

Reply:
Yes. But we can add them to one of the files in the binder subfolder. Can you point out which should not be included?

Reply:
Already removed all the ones you added. Anything that is not conda installable cannot appear here, so for now I've removed pydeck, pyecharts, idom, xlsxwriter and pyarrow. You could readd pyarrow since that is available.